kornia/augmentation/_2d/intensity/gamma.py

```diff
@@ -0,0 +1,83 @@
+from kornia.augmentation._2d.intensity.base import IntensityAugmentationBase2D
+from kornia.enhance.adjust import adjust_gamma
+
+
+class RandomGamma(IntensityAugmentationBase2D):
+    r"""Apply a random transformation to the gamma of a tensor image.
+
+    This implementation aligns PIL. Hence, the output is close to TorchVision.
+
+    .. image:: _static/img/ColorJitter.png
+
+    Args:
+        p: probability of applying the transformation.
+        gamma: the gamma factor to apply
+        gain: the gain factor to apply
+        silence_instantiation_warning: if True, silence the warning at instantiation.
+        same_on_batch: apply the same transformation across the batch.
+        keepdim: whether to keep the output shape the same as input (True) or broadcast it
+                 to the batch form (False).
+    Shape:
+        - Input: :math:`(C, H, W)` or :math:`(B, C, H, W)`, Optional: :math:`(B, 3, 3)`
+        - Output: :math:`(B, C, H, W)`
+
+    .. note::
+        This function internally uses :func:`kornia.enhance.adjust_gamma`
+
+    Examples:
+        >>> rng = torch.manual_seed(0)
+        >>> inputs = torch.rand(1, 3, 3, 3)
+        >>> aug = RandomGamma([0.5,2.],[1.5,1.5] p=1.)
+        >>> aug(inputs)
+        tensor([[[[1.0000, 1.0000, 0.3912],
+                  [0.4883, 0.7801, 1.0000],
+                  [1.0000, 1.0000, 0.9702],
+        <BLANKLINE>
+                 [[1.0000, 0.8368, 0.9048],
+                  [0.1824, 0.5597, 0.7609],
+                  [1.0000, 1.0000, 1.0000]],
+        <BLANKLINE>
+                 [[0.5452, 0.7441, 1.0000],
+                  [1.0000, 0.8990, 1.0000],
+                  [0.9267, 1.0000, 1.0000]]]])
+
+    To apply the exact augmenation again, you may take the advantage of the previous parameter state:
+        >>> input = torch.randn(1, 3, 32, 32)
+        >>> aug = ColorJitter(0.1, 0.1, 0.1, 0.1, p=1.)
+        >>> (aug(input) == aug(input, params=aug._params)).all()
+        tensor(True)
+    """
+
+    def __init__(
+        self,
+        gamma: Union[Tensor, float, Tuple[float, float], List[float]] = 0.0,
+        gain: Union[Tensor, float, Tuple[float, float], List[float]] = 0.0,
+        same_on_batch: bool = False,
+        p: float = 1.0,
+        keepdim: bool = False,
+        return_transform: Optional[bool] = None,
+        silence_instantiation_warning: bool = False,
+    ) -> None:
+        super().__init__(p=p, return_transform=return_transform, same_on_batch=same_on_batch, keepdim=keepdim)
+
+        if not silence_instantiation_warning:
+            warnings.warn(
+                "`ColorJitter` is now following Torchvision implementation. Old "
+                "behavior can be retrieved by instantiating `ColorJiggle`.",
+                category=DeprecationWarning,
+            )
+
+        self.gamma = gamma
+        self.gain = gain
+        self._param_generator = cast(rg.RandomGammaGenerator, rg.RandomGammaGenerator(gamma, gain))
```

Comment:
probably you could use directly a `PlainUniformGenerator`
https://github.com/kornia/kornia/blob/master/kornia/augmentation/_2d/intensity/plasma.py#L50

/cc @shijianjian